### PR TITLE
ClientAI - CoreClient

### PR DESCRIPTION
**Pull request by 31Nathan**
Add a Core Client object
This will be the main object for the client
The connectToServer() method tries to connect the client to the server with the given parameter

### CODE REVIEW

**Review by Cavonstavant — APPROVED** [review]
all good

**Review by pariciadra — APPROVED** [review]
LGTM

File: src/zappy_ai_src/ai_core.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        try:
+            self.__mapX = int(infos[0])
+            self.__mapY = int(infos[1])
+            self.__placesLeft = int(infos[2])
+        except ValueError:
+            print("The values of the GetRepJoinTeam() aren't integers", file=stderr)
+            return False
+        return True
+
+    def run(self) -> None:
```
Comment (pariciadra) [inline src/zappy_ai_src/ai_core.py:48]: This function don't do anything ?

**Review by pariciadra — COMMENTED** [review]
Pls check the little comment

**Review by GuyomT — APPROVED** [review]
LGTM

**Pull request merged**